tests/integration/test_integration.py

```diff
@@ -747,19 +748,14 @@ def test_downgrade(encoding, monkeypatch):
 
 @allencodings
 def test_span_tags(encoding, monkeypatch):
-    monkeypatch.setenv("DD_TRACE_API_VERSION", encoding)
-
-    t = Tracer()
-    with mock.patch("ddtrace.internal.writer.writer.log") as log:
-        s = t.trace("operation", service="my-svc")
+    with snapshot_context(token="tests.integration.test_integration.test_span_tags"):
+        monkeypatch.setenv("DD_TRACE_API_VERSION", encoding)
+        s = ddtrace.tracer.trace("operation", service="my-svc")
```

Comment:
this still needs to be updated to use `t = Tracer()` again, otherwise the env var isn't being used at all.

Reply:
The snapshot tests only capture traces generated by the global tracer. I had to convert this test to a subprocess test.

Also, traces generated by v0.4 and v0.5 encodings generate different snapshots. The v0.5 encoding adds a `type` field to all spans. This kinda surprised me.

Reply:
oh.... makes sense on both accounts